.github/workflows/proto-upgrade.yml

```diff
@@ -0,0 +1,52 @@
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: Flipt release version tag to generate
+        required: true
+
+permissions:
+  contents: write
+  pull-requests: write
+
+name: proto-upgrade
+jobs:
+  proto-upgrade:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+
+      - name: Clone Flipt into sub-directory
+        run: gh repo clone flipt-io/flipt
+        env:
+          GH_TOKEN: ${{ github.token }}
```

Comment:
I think its `secrets.GITHUB_TOKEN`

Reply:
This was actually suggested in an error by github. Is it a new or outdated thing perhaps?

Reply:
weird, never seen that https://docs.github.com/en/actions/security-guides/automatic-token-authentication